python_modules/libraries/dagster-fivetran/dagster_fivetran/asset_defs.py

```diff
@@ -51,6 +51,28 @@
 logger = get_dagster_logger()
 
 
+class DagsterFivetranTranslatorProps(NamedTuple):
+    connector_id: str
+    table_name: str
+    io_manager_key: Optional[str]
```

Comment:
I wouldn't worry about threading this through - a user can choose to do that in their own translator subclass, but we're generally deprecating IO managers in favor of other ways of loading data